### ISSUE TITLE
Who not supporting match_phrase query?

### ISSUE THREAD

**Issue by tottishi05** [mobz/elasticsearch-head#280]
In our product environment, **match_phrase** query is used frequently

**Comment by philipskokoh** [comment]
You can use last tab (any request) for match_phrase query

**Comment by tottishi05** [comment]
@philipskokoh 
Yes, I know. But why this useful search type is not supported in the third tab?

**Comment by philipskokoh** [comment]
Currently, no resources to work on that feature. Do you want to contribute? :)

**Comment by tottishi05** [comment]
I really want to contribute to this project soon

**Comment by philipskokoh** [comment]
feel free to submit PR for this feature
